habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -437,14 +616,16 @@ def calculate_reach_pose(self, obj_pos: mn.Vector3, index_hand=0):
         )
         relative_pos = inv_T.transform_vector(obj_pos - root_pos)
 
-        curr_poses, curr_transform = self._trilinear_interpolate_pose(
-            mn.Vector3(relative_pos), hand_data
-        )
+        # TODO
+        if hand_data is not None:
```

Comment:
Here, we ignore updating hand poses if we select an animation file that does not contain hand data (such as the default one).